### DISCUSSION TITLE
Mode nuit

### DISCUSSION THREAD

**Discussion by gastibelza** [marian-m12l/studio#481] [Ideas]
Merci et bravo pour ce superbe programme !
Serait-il possible d'implémenter la modification des paramètres du mode nuit ?

Merci :)

**Comment by marian-m12l** [comment]
Merci pour le retour :)
Vous parlez de la personnalisation du mode nuit (volume, ...) ? 
À ma connaissance il s'agit d'un paramétrage global pour l'appareil, et non selon le pack d'histoires. Au niveau d'un pack on peut indiquer s'il est compatible ou non.
Pour gérer les paramètres au niveau de l'appareil, il faut passer par le luniistore : https://support.lunii.com/hc/fr/articles/4404827783697-Comment-personnaliser-mon-Mode-Nuit
